pokemongo_bot/cell_workers/catch_visible_pokemon.py

```diff
@@ -14,9 +14,9 @@ def work(self):
                 key=
                 lambda x: distance(self.bot.position[0], self.bot.position[1], x['latitude'], x['longitude'])
             )
-
+            user_web_catchable = 'web/catchable-%s.json' % (self.bot.config.username)
```

Comment:
I think we like using `.format`

`user_web_catchable = 'web/catchable-{}.json'.format(self.bot.config.username)`